gkShell/config/konfig-mysql.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Same as bellow, and more generally for each file a user is able to update for personal config, but changes  are not necessarily needed to be pushed.

Reply:
done using docker one